tests/net/6lo/src/main.c

```diff
@@ -397,7 +396,6 @@ static struct net_6lo_data test_data_1 = {
 	.ipv6.vtc = 0x60,
 	.ipv6.tcflow = 0x00,
 	.ipv6.flow = 0x00,
-	.ipv6.len = { 0x00, 0x00 },
```

Comment:
Why here you remove this line completely, while bellow replace with ".ipv6.len = 0,".